packages/vscode-extension-dashbuilder-editor/src/extension/ComponentsHttpServer.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { LocalHttpServer } from "@kie-tools-core/backend/dist/api";
+import { getPortPromise } from "portfinder";
+import * as http from "http";
+import * as fs from "fs";
+import * as path from "path";
+
+export class ComponentServer extends LocalHttpServer {
+  private componentsPath: string;
+  private server: http.Server;
+
+  requestListener = (request: any, response: any) => {
+    if (request.url == "/" || request.url == "" || request.url == "index.html") {
```

Comment:
```suggestion
    if (["/", "", "index.html"].includes(request.url)) {
```
Optional to make it shorter.

Reply:
TS HACKER!